infercnvpy/tl/_copykat.py

```diff
@@ -137,13 +149,16 @@ def copykat(
     }
 
     # Drop cols
-    new_cpkat = copyKAT_result.drop(["chrom", "chrompos", "abspos"], axis=1).values
-
+    new_cpkat = copyKAT_result.drop(["chrom", "chrompos", "abspos"], axis=1)
+    # align cells
+    new_cpkat = new_cpkat.loc[:,adata.obs.index]
+    copyKAT_pred = adata.obs.merge(copyKAT_pred,left_index=True,right_index=True,how='left')['copykat.pred']
```

Comment:
I don't think this line is necessary. If you assign a pandas series with index to a dataframe, it will automatically reorder it such that it matches the index.

Reply:
that's good to know, thanks for the suggestion!